runner/cf-driver-go/README.md

```diff
@@ -0,0 +1,51 @@
+# CloudFoundry (cloud.gov) driver
+
+This is a _driver_ developed for use with `gitlab-runner`’s [Custom executor](https://docs.gitlab.com/runner/executors/custom.html) to prepare, run, and clean up runner managers, workers, and services.
+
+## Go setup
+
+### Install Go
+
+You can use Homebrew to install Go on MacOS (it is generally unnecessary to use a version manager). You could also download Go packages for Linux, Mac, and Windows from [Go's website](https://go.dev/doc/install).
```

Comment:
[question] why is a version manager generally unnecessary? if go.mod is locking us down to a specific path level then I'd think that just using homebrew for versions would introduce pain. Or is it fine since we're a small team using the same set of tools?

Reply:
It's because the language has a backwards compatibility guarantee. There isn't ever going to be a Go 2. So you might want to update at some point but you won't need to downgrade. There _are_ version managers, and we _could_ use one, but I'm not imagining a scenario where that would make sense.

Reply:
Perhaps! Though, they _do_ do some kind of funky stuff with environment flags to mitigate difficulties that come up changing things, and I think some of it might be automatic based on the version you specify in `go.mod`.

Anyway, seems like it couldn't be worse than JavaScript :-)